concepticondata/concepticon.tsv

```diff
@@ -3967,3 +3967,5 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 3981	GUAVA	The physical world	Fruit from the myrtle family, native to Latin America.	Person/Thing	
 3982	BREEZE	The physical world	A gentle to moderate wind.	Person/Thing	
 3983	RED DEER	Animals	The red deer (Cervus elaphus) is one of the largest deer species. A male red deer is called a stag or hart, and a female is called a hind. Red deer are ruminants, characterized by a four-chambered stomach. Genetic evidence indicates that the red deer, as traditionally defined, is a species group, rather than a single species, though exactly how many species the group includes remains disputed.	Person/Thing	
+3984	THESE (PLURAL OF THIS)	Miscellaneous function words	The indicated objects, items, etc. close to the speaker.	Other	
+3985	THOSE (PLURAL OF THAT)	Miscellaneous function words	The indicated objects, items, etc. distant from the speaker.	Other	
```

Comment:
Thanks for adding these concepts and for updating them in the lists. There's a couple of more entries that could be updated. Can I add this to your PR? This would be:

- BirketSmith-1928-510-392
- Dolch-1936-220-38
- Hale-1973-1798-1788
- Janhunan-2008-235-19
- Dolch-1936-220-41
- Hale-1973-1798-1789
- Janhunan-2008-235-22

If you're okay with that I can update with a commit to your PR.

Reply:
Ah, that would be perfect, since I won't have time!

Reply:
Dunno why I missed these, I ran the check and did not find the glosses. Maybe I should've also searched for real...

Reply:
Done! The diff for Hale is a bit unwieldy because there was in issue with the column formatting which I've fixed. Should have been a separate commit, sorry. :)